CMakeLists.txt

```diff
@@ -110,6 +91,11 @@ set(CMAKE_C_VISIBILITY_INLINES_HIDDEN YES)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 
+set(CMAKE_POSITION_INDEPENDENT_CODE YES)
+if(NOT APPLE)
+  set(CMAKE_INSTALL_RPATH "$ORIGIN")
```

Comment:
What if I want to override the RPATH?

Reply:
Maybe something like this?
```cmake
if(NOT APPLE AND NOT DEFINED CMAKE_INSTALL_RPATH)
  set(CMAKE_INSTALL_RPATH "$ORIGIN")
endif()
```

What is the CMake default? Nothing?

Alternatively, maybe something like this?
```cmake
if(NOT APPLE)
  list(APPEND CMAKE_INSTALL_RPATH "$ORIGIN")
endif()
```

I think I prefer the former though.

No, it wasn't previously available, but as long as we're fixing all the things, might as well fix this too.